src/phoenix/trace/exporter.py

```diff
@@ -30,6 +32,80 @@ def export(self, _: Any) -> None:
         pass
 
 
+class OTELNoOpExporter(SpanExporter):
+    def export(self, _: Any) -> SpanExportResult:
+        return SpanExportResult.SUCCESS
+
+    def shutdown(self) -> None:
+        pass
+
+
+class _OpenInferenceExporter:
+    def __init__(
+        self,
+        endpoint: Optional[str] = None,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+        no_op: bool = False,
+    ) -> None:
+        """
+        A compatibility class used to configure the Phoenix endpoint for instrumentation.
+
+        Parameters
+        ----------
+        endpoint: Optional[str]
+            The endpoint of the Phoenix server (collector). This should be set if the Phoenix
+            server is running on a remote instance. It can also be set using environment
+            variable `PHOENIX_COLLECTOR_ENDPOINT`, otherwise it defaults to `http://127.0.0.1:6006`
+            Note, this parameter supersedes `host` and `port`.
+        host: Optional[str]
+            The host of the Phoenix server. It can also be set using environment
+            variable `PHOENIX_HOST`, otherwise it defaults to `127.0.0.1`.
+        port: Optional[int]
+            The port of the Phoenix server. It can also be set using environment
+            variable `PHOENIX_PORT`, otherwise it defaults to `6006`.
+        no_op: bool
+            If enabled, the exporter will not send any data to Phoenix. This is useful for
+            testing and development.
+        """
+        self._host = host or get_env_host()
+        self._port = port or get_env_port()
+        endpoint = endpoint or get_env_collector_endpoint() or f"http://{self._host}:{self._port}"
+        # Make sure the url does not end with a slash
+        self._base_url = self._phoenix_otel_endpoint(endpoint.rstrip("/"))
+        self._warn_if_phoenix_is_not_running()
+
+        self.otel_exporter: SpanExporter = (
+            OTELNoOpExporter() if no_op else OTLPSpanExporter(endpoint=self._base_url)
+        )
+
+    def _phoenix_otel_endpoint(self, url: str) -> str:
+        return f"{url}/v1/traces"
+
+    def _warn_if_phoenix_is_not_running(self) -> None:
+        try:
+            requests.get(f"{self._base_url}/arize_phoenix_version").raise_for_status()
+        except Exception:
+            logger.warning(
+                f"Arize Phoenix is not running on {self._base_url}. Launch Phoenix "
+                f"with `import phoenix as px; px.launch_app()`"
+            )
+
+    @classmethod
+    def _from_legacy_exporter(
```

Comment:
This conversion also seems necessary because the old exporter can be simply discarded, and it has no relation to the new OTEL exporters.

Reply:
addressed in 4aaaac76